tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,625 @@
+        content  = data.template_file.kubeconfig.rendered
+        filename = "${path.module}/kubeconfig"
+    }
+    ```
+
+6. Initialize your Terraform configuration and apply it to create your resources.
+    ```
+    terraform init
+    terraform apply
+    ```
+
+7. Configure your `KUBECONFIG` environment variable to point to the `kubeconfig` file you just created via Terraform
+    ```
+    export KUBECONFIG=kubeconfig
+    ```
+
+8. List your nodes now to check that your cluster is ready, and that there are two nodes per availability zone.
+    ```
+    kubectl get nodes
+
+    NAME                                             STATUS   ROLES    AGE   VERSION
+    scw-kapsule-multi-az-pool-fr-par-1-61e22198f8c   Ready    <none>   89s   v1.28.0
+    scw-kapsule-multi-az-pool-fr-par-1-8334e772ced   Ready    <none>   82s   v1.28.0
+    scw-kapsule-multi-az-pool-fr-par-2-1bcf90f3683   Ready    <none>   90s   v1.28.0
+    scw-kapsule-multi-az-pool-fr-par-2-33265e85597   Ready    <none>   86s   v1.28.0
+    scw-kapsule-multi-az-pool-fr-par-3-44b14b7bbbd   Ready    <none>   84s   v1.28.0
+    scw-kapsule-multi-az-pool-fr-par-3-863491657c7   Ready    <none>   80s   v1.28.0
+    ```
+
+## Nginx Ingress Controller as a stateless multi-az application
+
+A very common pattern is to deploy an Ingress Controller to externally expose applications hosted inside the Kapsule cluster. Configuring a multi-AZ deployment of your Ingress Controller will greatly improve its resilience and scalability.
+
+In this part we will deploy an [ingress-nginx](https://github.com/kubernetes/ingress-nginx) in a multi-AZ, production ready, configuration, using the previously created Kapsule cluster. The official Helm for the nginx controller is full-featured, so we will use it through the Terraform Helm provider.
+
+* Healthy cluster
+<Lightbox src="scaleway-k8s-kapsule-multi-az-2.webp" alt="" size="large" />
+
+* Loss of one AZ
+<Lightbox src="scaleway-k8s-kapsule-multi-az-3.webp" alt="" size="large" />
+
+<Message type="note">
+    For this section you need, as a prerequisite, a domain managed on the Scaleway "Domains and DNS" service.
+</Message>
+
+The domain userd for this tutorial can be of type `internal` when bought with your Scaleway account, or of the `external` type if you want to re-use a domain bought with an external DNS registrar. If you choose the `external` method, you can follow the [How to add an external domain to DNS](https://www.scaleway.com/en/docs/network/domains-and-dns/how-to/add-external-domain/). If you don't want to delegate all your domain to your Scaleway account, import your domain but don't change the name servers in your registrar. Then you can create a "DNS zone" in Scaleway on your external domain, and declare two NS records in your registrar matching this zone. For example, with a domain "your-domain.tld" imported in Scaleway, you can create a DNS zone "scw", and delegate this zone in your registrar by adding those two records:
```

Comment:
This sentence makes no sense

- If you choose the `external` method, you can follow the "How to add an external domain to DNS".

Perhaps missing one word? 
- e.g you can follow the guide "How to..."